js/build.js

```diff
@@ -22,12 +22,14 @@ Fliplet.Widget.instance('list', function(data) {
 
   var swipeToSaveLabel = data.swipeToSaveLabel || T('widgets.list.noImages.defaultListName');
 
-  $container.translate({ swipeToSaveLabel: swipeToSaveLabel });
-
-  if (data.swipeToSave) {
-    window.ui = window.ui || {};
-    window.ui['swipeSavedList' + $container.attr('data-list-uuid')] = new SwipeSaveList(this, {
-      savedListLabel: swipeToSaveLabel
-    });
-  }
+  Fliplet().then(function() {
+    $container.translate({ swipeToSaveLabel: swipeToSaveLabel });
+
+    if (data.swipeToSave) {
+      window.ui = window.ui || {};
+      window.ui['swipeSavedList' + $container.attr('data-list-uuid')] = new SwipeSaveList(this, {
```

Comment:
@skrupnyk The `this` variable in this line is likely referencing the wrong element, now that you've changed the context with `Fliplet().then(function() {})`